src/components/Create/Create.jsx

```diff
@@ -172,64 +179,74 @@ function Create(props) {
             setLevelText(i18n.t("Level required"))
             return
         }
+        const catchmentNames = jobs?.find((name) => name.name.toLowerCase() === formInputs.name.toLowerCase())
+        // crosscut was prepended to published catchments
+        const publishedNames = currentNames.find((name) => name.name.toLowerCase().split("crosscut ")[1] === formInputs.name.toLowerCase())
+
+        if (publishedNames !== undefined || catchmentNames !== undefined) return setNameText(i18n.t("Name is already in use"))
+
         
         setIsLoading(true)
-        try {
-            const resp = await createCatchmentJob(formInputs).catch( async (err) => {
-                const data = JSON.parse(await err.response.text())
-                if (data.csv) {
-                    const resp = papaparse.parse(data.csv.trim(), { header: true })
-                    return { error: resp }
+        await createCatchmentJob(formInputs).then(() => {
```

Comment:
It looks like the same site list is used regardless of what Country I choose from the drop down (it just shows that all sites are outside of the boundary if I choose something other than SLE). When retrieving the site list when a user clicks Create, can we limit sites based on Country as well? And provide an error if the Country isn't in the DHIS2 instance?

Reply:
Let's handle this in another issue: https://github.com/crosscutio/dhis2-crosscut-app/issues/26